### PR TITLE
Update type names in v9 API

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: common/api-review/performance-exp.api.md [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
-}
+export function getPerformance(app?: FirebaseApp): PerformanceMonitoring;
 
 // @public
-export function getPerformance(app?: FirebaseApp): FirebasePerformance;
+export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): PerformanceMonitoring;
 
 // @public
-export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+export interface PerformanceMonitoring {
```
Comment (jposuna) [inline common/api-review/performance-exp.api.md:16]: Just curious about why we have this below. Wouldn't we want to "introduce it" before the function that uses it?

**Review by jposuna — APPROVED** [review]
LGTM for Fireperf

File: common/api-review/performance-exp.api.md (hunk 1 above)
Comment (Feiyang1) [reply common/api-review/performance-exp.api.md]: This file is auto generated by api-extractor to describe APIs in human readable format. You can see that it's a md file, not actual code where the order also doesn't matter. Types are compile time artifacts, and are available to things before defined them in the file.

Comment (Feiyang1) [reply common/api-review/performance-exp.api.md]: BTW, I reverted Perf related changes because we decided to keep using `FirebasePerformance` as the name.

Comment (jposuna) [reply common/api-review/performance-exp.api.md]: Sounds good, thanks!

**Review by hsubox76 — APPROVED** [review]

**Review by zwu52 — APPROVED** [review]

**Review by egilmorez — APPROVED** [review]
LG.  We'll need to change a bunch of links from the guides and I can take that action as soon as reference docs are generated from this source.  Thanks!

**Pull request merged**